zingolib/src/wallet/data.rs

```diff
@@ -783,7 +788,96 @@ pub mod summaries {
         }
     }
 
-    /// TODO: doc comment
+    /// Basic transaction summary interface
+    pub trait TransactionSummaryInterface {
```

Comment:
Why is this logic packaged in a separate trait?   Why aren't these methods inherent to a `TransactionSummary`?

Reply:
I am fine with it as is, just curious.